src/modules/admin/layouts/AdminLayout/AdminLayout.tsx

```diff
@@ -0,0 +1,64 @@
+import { useTranslation } from 'next-i18next';
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+import { FC, useEffect, useMemo } from 'react';
+import { useDispatch } from 'react-redux';
+
+import { ListNavigationItem } from '@shared/components';
+import SidebarLayout from '@shared/layouts/SidebarLayout/SidebarLayout';
+import { setShowZendesk } from '@shared/store/ui';
+
+import { ADMIN_NAVIGATION_LINKS } from '../../const';
+
+import { AdminLayoutProps } from './AdminLayout.types';
+
+const AdminLayout: FC<AdminLayoutProps> = ({ children, className, contentTitle }) => {
+	const { asPath } = useRouter();
+	const dispatch = useDispatch();
+
+	const sidebarLinks: ListNavigationItem[] = useMemo(
+		() =>
+			ADMIN_NAVIGATION_LINKS().map(({ id, label, href, children }) => ({
+				id,
+				node: ({ linkClassName }) => (
+					<Link href={href}>
+						<a className={linkClassName} title={label}>
+							{label}
+						</a>
+					</Link>
+				),
+				active: href ? asPath.includes(href) : false,
+				children: children?.().map(({ id, label, href }) => ({
+					id,
+					node: ({ linkClassName }) => (
+						<Link href={href}>
+							<a className={linkClassName} title={label}>
+								{label}
+							</a>
+						</Link>
+					),
+					active: href ? asPath.includes(href) : false,
+				})),
+			})),
+		[asPath]
+	);
+
+	const { t } = useTranslation();
+
+	useEffect(() => {
+		dispatch(setShowZendesk(false));
+	}, [dispatch]);
```

Comment:
PREFERENCE: i keep all hooks and useEffect code at the top of the component
then the functions
and then the render functions and final return statement